docs/topics/compiler-reference.md

```diff
@@ -156,6 +156,12 @@ environment.
 
 Use a custom JDK home directory to include into the classpath if it differs from the default `JAVA_HOME`.
 
+### -jdk-release
```

Comment:
"-X" is just a prefix for options which are **non-standard**, i.e. there are no guarantees how they work and if they are present in the new release at all. There are many such options, in fact many more than standard options, see https://github.com/JetBrains/kotlin/blob/master/compiler/testData/cli/jvm/extraHelp.out (as opposed to https://github.com/JetBrains/kotlin/blob/master/compiler/testData/cli/jvm/help.out).

It looks like we don't normally document non-standard options just because we don't want users to expect that they are stable and can be relied on. So by this logic, we don't _have_ to document `-Xjdk-release`. But maybe in this specific case, it's better to document it for several reasons:
1) the issue [KT-29974](https://youtrack.jetbrains.com/issue/KT-29974) had many votes
2) we'll likely make the option stable in a future release and it'll look exactly the same without "-X", e.g. `-jdk-release 11`
3) IMHO we need to improve documentation around -X options if we want feedback from users who are enabling them (there's a related issue [KT-52122](https://youtrack.jetbrains.com/issue/KT-52122))

Reply:
Thank you!
Ok, I'll add `X` to this specific option.
> IMHO we need to improve documentation around -X options if we want feedback from users who are enabling them (there's a related issue [KT-52122](https://youtrack.jetbrains.com/issue/KT-52122))

@koshachy and I know about this task.